app/client/src/components/pages/Community/components/tabs/Protect.js

```diff
@@ -233,6 +233,7 @@ function Protect() {
   // normalize attains plans data with grts projects data
   useEffect(() => {
     if (attainsPlans.status === 'fetching') return;
+    if (!attainsPlans.data?.items?.length) return;
```

Comment:
@cschwinderg thanks for talking this through... I'm going to remove the optional chaining accessor operators and only check that the items property exists on the data object – only to make it a little more consistent with what we're checking for in other components.

But to be clear, what you have works (and is even more thorough, as it checks that the items object (really an array) has a length attribute as well